mon-pix/tests/unit/components/routes/login-form_test.js

```diff
@@ -58,8 +61,56 @@ module('Unit | Component | routes/login-form', function (hooks) {
         await component.authenticate(eventStub);
 
         // then
-        assert.true(component.isErrorMessagePresent);
-        assert.false(component.hasUpdateUserError);
+        assert.ok(component.errorMessage);
+      });
+
+      module('when user is temporary blocked', function () {
+        test('set right error message', async function (assert) {
```

Comment:
**nitpick**

```suggestion
        test('sets the correct error message', async function (assert) {
```

car :

> “Correct” implies something which is absolutely true and has no errors, while “right” implies something which is merely an opinion.

https://allthedifferences.com/difference-between-is-that-right-and-is-that-correct/#:~:text=Correct%20is%20free%20from%20error,which%20is%20merely%20an%20opinion.